### DISCUSSION TITLE
Device Level Options

### DISCUSSION THREAD

**Discussion by pineconefox** [tsightler/ring-mqtt#416] [General Discussion]
Hello, I am trying to get snapshots out of my ring wired doorbell. In the wiki "Device Level Options" are mentioned. Where can i configure them?

**Comment by tsightler** [comment]
If you are talking about in Home Assistant, then just find the device in HA devices tab and all the controls and sensors for that device are there.  If you are using MQTT directly then you must use the MQTT topic to change the setting for the requested device.
